### PR TITLE
fix: applying new headline button guideline

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- New guideline for button component.
- Applied the new component when the user has filtering applied.
- Tooltip is removed as it is no longer necessary since this is now a button with its own label.

Preview: 
image

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-538 #done
WT-539 #done

### CODE REVIEW

File: packages/shared/src/styles/components/buttons.css [hunk 1]
```diff
@@ -31,6 +31,14 @@
     border-radius: 0.875rem;
   }
 
+  &.headline {
+    height: 3rem;
+    padding: 0 1.5rem;
+    font-size: 1.0625rem;
+    border-radius: 0.875rem;
+    color: var(--theme-label-primary);
+  }
```
Comment (rebelchris) [inline packages/shared/src/styles/components/buttons.css:40]: Is this a recurring thing?
Why don't we set them as classNames?

File: packages/shared/src/components/MainFeedLayout.tsx [hunk 2]
```diff
@@ -254,6 +240,23 @@ export default function MainFeedLayout({
 
   const hasFiltered = feedName === MainFeedPage.MyFeed && !alerts?.filter;
 
+  /* eslint-disable react/no-children-prop */
+  const feedHeading = {
+    [MainFeedPage.MyFeed]: !hasFiltered ? (
```
Comment (rebelchris) [inline packages/shared/src/components/MainFeedLayout.tsx:245]: Optionally maybe we can create a MyFeedHeading component which will do the conditional rendering? (Just to clean this up some more)

**Review by rebelchris — APPROVED** [review]
Nice work! 

Just some small comments, no breaking from my side.

File: packages/shared/src/components/MainFeedLayout.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/MainFeedLayout.tsx:245]: Good point. Will make this part cleaner. Let me refactor this one.

File: packages/shared/src/styles/components/buttons.css (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/styles/components/buttons.css:40]: It is due to using `primary`, `secondary`, etc., automatically applying their own color of the label, this is to force it, regardless of which type of button was used.

**Review by rebelchris — APPROVED** [review]
Thanks for the refactor.

**Pull request merged**